lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -232,6 +235,21 @@ def get(self, request, username):
 
         return Response(user_certs)
 
+    def _viewable_by_requestor(self, request, username):
```

Comment:
Future: we should start using OEP-9's recommendation of Django Rules at some point.